pages/index.js

```diff
@@ -1,11 +1,14 @@
 import Heading from "../components/Heading";
 import SubHeading from "../components/SubHeading";
+import DogList from "../components/DogList";
 
 export default function Home() {
   return (
     <main>
-      <Heading>🐬 Capstone Template 🐬</Heading>
-      <SubHeading>We dolphinitely love coding!</SubHeading>
+      <Heading>Dog Pass 🐕‍🦺</Heading>
+      <SubHeading></SubHeading>
+      <br></br>
```

Comment:
The `<br>` element is not best practise since you could style your components with styled components (don't forget to import it with `import styled from "styled-components";`), e.g. you could give your `DogsList` component a little padding-top, and you see the same effect as a `br` tag